<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePassword</class>
 <widget class="QDialog" name="ChangePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>219</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>303</width>
    <height>219</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>303</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>303</width>
    <height>219</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>EduTest Change Password</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Logo.qrc">
    <normaloff>:/logo/WindowLogo.png</normaloff>:/logo/WindowLogo.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="Info">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>10</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Придумайте новый пароль, он может состоять
из латиницы, цифр и спецсимволов (кроме +)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewPasswordLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Новый пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="ChangePasswordMessage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>140</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Здесь выводятся сообщения</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OldPasswordLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Старый пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewPasswordRepeatLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Повторите новый пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChangePasswordButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить пароль</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>OldPasswordLine</tabstop>
  <tabstop>NewPasswordLine</tabstop>
  <tabstop>NewPasswordRepeatLine</tabstop>
  <tabstop>ChangePasswordButton</tabstop>
  <tabstop>CancelButton</tabstop>
 </tabstops>
 <resources>
  <include location="Logo.qrc"/>
 </resources>
 <connections/>
</ui>
